content/_code-samples/checks/py/account_checks.py

```diff
@@ -0,0 +1,100 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models import AccountObjects
+from xrpl.utils import drops_to_xrp, hex_to_str, ripple_time_to_datetime
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234") # connect to the testnetwork
+
+
+# Query the ledger for all xrp checks an account has created or received
+
+# wallet address to query
+wallet_addr = "rPKcw5cXUtREMgsQZqSLkxJTfpwMGg7WcP"
+
+# dict to store all the checks
```

Comment:
Please remove comments which just repeat what the code says - if they help explain the intent of a code block, that's fine, but in general aim to have the name of the variable be enough context to understand the intent of the variable.

Also please capitalize the first letter of comments